academic-services/knowledge-exploration-service/how-to-deploy-new-indexes-to-makes.md

```diff
@@ -0,0 +1,175 @@
+
+3. Select **Endpoints** on the left hand side of the page to open the list of endpoints for this traffic manager profile.
+
+![Select the "Endpoints" item on the left hand side](media/how-to-deploy-new-indexes-to-makes-select-endpoints.png)
+
+4. Select **Add** at the top of the screen to add a new endpoint.
+
+![Select Add to create an endpoint](media/how-to-deploy-new-indexes-to-makes-click-add-endpoint.png)
+
+3. Apply the following settings:
+
+    | Setting | Value |
+    | --------| ----- |
+    | Type | Azure endpoint |
+    | Name | *Add a descriptive name of your choosing here* |
+    | Target resource type | Public IP Address |
+    | Target resource | Select the name of the public IP address that was created for you MAKES deployment |
+    | Custom Header settings | **Leave Blank**  |
+    | Add as disabled | **checked** |
+
+9. Select **Ok** to create your new endpoint.
+
+It will take Azure a moment to create your new endpoint, but once completed it will be added to the list of endpoints available in your Traffic Manager Profile.
+
+## Swap MAKES instances in Traffic Manager
+
+When you are ready to bring new traffic to your new MAKES instance you must enable the new instance and disable the old.
+
+1. From the endpoints section of your Traffic Manager Profile, select the endpoint that you just created above.
+
+2. Move the status toggle from **Disabled** to **Enabled** by clicking on it.
+
+3. Select **Save** at the top to save your changes.
+
+4. Select on the old endpoint that was serving production traffic.
+
+5. Move the status toggle from **Enabled** to **Disabled** by clicking on it.
+
+6. Select **Save** at the top to save your changes.
+
+## Verify your service is working as expected
+
+In the list of enpoints, watch the **Monitor Status** of the new endpoint you created.  In the beginning it will say **Checking Endpoint**.  Click the Refresh button on the top every 30 secs or so until the status changes to **Online**; this could take a few minutes.
+
+Once the new endpoint monitor is in the **Online** status, verify the API is up and running.  In the **Overview** section of your Traffic Manager Profile, there will be a URL next to the **DNS Name** property.  Copy this value and paste it into a browser of your choice.  Ex: **http://<your_traffic_manager_profile_name>.trafficmanager.net**.  Verify the instance is working by running some queries or use your favorite tool to execute queries against the API endpoints and verify the responses and response codes.  If anything is not working as expected, reverse the steps above to enable the old version of the API.
+
+> [!NOTE]
+> Traffic Manager by default uses http, not https
+
+## Clean up resources 
+
+Once you have verified that your new MAKES instance is performing as expected, you can safely delete the resources from your previous MAKES deployment.
```

Comment:
Add the following:
For more information on how to delete a MAKES deployment, see [Delete a MAKES deployment](how-to-delete-api-instances.md)